django_filters/rest_framework/backends.py

```diff
@@ -62,3 +62,10 @@ def to_html(self, request, queryset, view):
         }
         template = loader.get_template(self.template)
         return compat.template_render(template, context)
+
+    def get_fields(self, view):
```

Comment:
@tomchristie - could you take a look at 291717f? Does this seem correct? Thanks for the help.